components/ResultsPageComponents/WorkArrangement/WorkArrangement.jsx

```diff
@@ -2,7 +2,23 @@ import { Text, Flex } from "@chakra-ui/react";
 import Traveler from "../../../public/images/work-arrangement-results-icons/traveler.svg";
 import Line from "../../../public/images/work-arrangement-results-icons/l-shape-line.svg";
 
-export default function WorkArrangement() {
+export default function WorkArrangement({ workMode }) {
+  const surveyData = {
+    "On-site": workMode.onsite || "",
+    WFH: workMode.wfh || "",
+    Hybrid: workMode.hybrid || "",
+  };
+
+  const maxCountWorkMode = Object.entries(surveyData).reduce((a, b) =>
+    a[1] > b[1] ? a : b
+  );
+  const minCountWorkMode = Object.entries(surveyData).reduce((a, b) =>
+    a[1] < b[1] ? a : b
+  );
+  const mediumCountWorkMode = Object.entries(surveyData).sort(
+    (a, b) => b[1] - a[1]
+  )[1];
```

Comment:
I think we can simplify the process of obtaining the percentages for the 3 work arrangements by providing more context in `data/2022-results.json`... (see comments in `data/2022-results.json`)